SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -807,6 +807,9 @@ void USpatialReceiver::DestroyActor(AActor* Actor, Worker_EntityId EntityId)
 	// TODO: fix this with working sets (UNR-411)
 	NetDriver->StartIgnoringAuthoritativeDestruction();
 
+	IncomingRPCs.DropForEntity(EntityId);
```

Comment:
I think this should go in USpatialActorChannel::Cleanup(), and only trigger on destroy and level unloaded.

Reply:
Okay, I've moved that there. I assume the one for outgoing RPCs is still fine to be here?

Reply:
No I think the, sender code should be moved to the same section, as the entity is only deleted in those instances.

Reply:
Makes sense, I've updated this now.